### DISCUSSION TITLE
Make Table editable

### DISCUSSION THREAD

**Discussion by PFernandezTorres** [mantinedev/mantine#5302] [Feature requests]
An editable Table, would make possible the option to edit a record directly on the Table instead of deleting that record and create the same record again. As filling text fields at the end become a tedious and repetitive task.

**Comment by rtivital** [comment]
You can use one of the community libraries – https://github.com/orgs/mantinedev/discussions/1057

**Reply by PFernandezTorres** [reply]
Okay, perfect. Thanks for your answer!!
